EPlast/EPlast.BLL/Services/Statistics/StatisticsService.cs

```diff
@@ -99,6 +100,24 @@ public async Task<IEnumerable<RegionStatistics>> GetRegionStatisticsAsync(IEnume
             return regionStatistics;
         }
 
+        public async Task<IEnumerable<MembersStatistic>> GetAllCitiesStatisticsAsync()
+        {
+            var annualReports = await _repositoryWrapper.AnnualReports.GetAllAsync(
+                    include: source => source
+                        .Include(ar => ar.Creator)
+                        .Include(ar => ar.MembersStatistic)
+                        .Include(ar => ar.Date)
+                        .Include(ar => ar.City)
+                            .ThenInclude(c => c. Region));
+
+            var membersStatistics = new List<MembersStatistic>();
+            foreach (var report in annualReports)
+            {
+                membersStatistics.Add(report.MembersStatistic);
+            }
+            return membersStatistics;
```

Comment:
```suggestion
            return annualReports.Select(report => report.MembersStatistic);
```
I'm not sure, but you can write something like that.

Why don`t you use DTO ? As I understand, Members Statistic from Data Access